tasks/devcontainer.py

```diff
@@ -0,0 +1,212 @@
+"""
+vscode namespaced tags
+
+Helpers for getting vscode set up nicely
+"""
+import json
+import os
+from collections import OrderedDict
+from pathlib import Path
+
+from invoke import task
+from invoke.exceptions import Exit
+from libs.common.color import color_message
+
+from tasks.build_tags import build_tags, filter_incompatible_tags, get_build_tags, get_default_build_tags
+from tasks.flavor import AgentFlavor
+
+VSCODE_DIR = ".vscode"
+VSCODE_FILE = "settings.json"
+DEVCONTAINER_DIR = ".devcontainer"
+DEVCONTAINER_FILE = "devcontainer.json"
+DEVCONTAINER_NAME = "datadog_agent_devcontainer"
+
+
+@task
+def set_buildtags(
+    _,
+    target="agent",
+    build_include=None,
+    build_exclude=None,
+    flavor=AgentFlavor.base.name,
+    arch='x64',
+):
+    """
+    Modifies devcontainer settings file for this project to include correct build tags
```

Comment:
I think it only modifies `.vscode/settings.json` which can be unrelated to devcontainer

Reply:
you're right, it has nothing to do with containers 🤦